source/_patterns/02-organisms/00-global/01-footer.mustache

```diff
@@ -82,7 +89,14 @@
 				</ul>
 			</div>
 
-			<div class="siteFooter-legal siteFooter-group">
+            {{#request_url_borrow}}
```

Comment:
Why is this in here twice? (It's also on line 16)

Reply:
@jedlin-kiva It's there twice because the English/Spanish links are required when viewing the mobile version. It's similar to the links for Privacy Policy and TOU.